helm/deploy/manager_prod.yaml

```diff
@@ -5,7 +5,6 @@ image:
 
 controllerImage:
   repository: docker.io/scylladb
-  tag: latest
```

Comment:
would it be better to add `<unspecified>`? so we get an explicit failure and not default to latest if the substitution breaks